bin/runtime/src/lib.rs

```diff
@@ -356,17 +356,14 @@ impl pallet_aleph::Config for Runtime {
     type NextSessionAuthorityProvider = Session;
 }
 
-#[cfg(feature = "liminal")]
```

Comment:
There are still two leftoverts in this file

![image](https://github.com/Cardinal-Cryptography/aleph-node/assets/3909333/d690db1a-3194-4b02-9dc7-6b4aa7125448)

Reply:
removed